fuzzamoto-ir/src/operation.rs

```diff
@@ -79,6 +80,9 @@ pub enum Operation {
     BuildOpReturnScripts,
     BuildPayToAnchor,
 
+    // cmpctblock building operations
+    BuildCmpctBlock,
```

Comment:
nit: since we're spelling it out everywhere else

```suggestion
    BuildCompactBlock,
```